common/lib/xmodule/xmodule/modulestore/exceptions.py

```diff
@@ -50,8 +50,11 @@ def __str__(self, *args, **kwargs):
         """
         Print info about what's duplicated
         """
-        return '{0.store}[{0.collection}] already has {0.element_id}'.format(
-            self, Exception.__str__(self, *args, **kwargs)
+        return "{store}[{collection}] already has {element_id} ({exception})".format(
+            store=self.store,
+            collection=self.collection,
+            element_id=self.element_id,
+            exception=Exception.__str__(self, *args, **kwargs),
```

Comment:
Do you know why this code is `Exception.__str__(self, *args, **kwargs)` instead of just `self`? The string formatting should automatically coerce the `self` argument to a string.

If you don't want to make this change, that's fine with me -- I'm in favor of keeping this pull request small and focused as much as possible. This just caught my eye, and I was confused.